lib/request.js

```diff
@@ -257,15 +257,14 @@ module.exports = {
    */
 
   get fresh() {
-    const method = this.method;
-    const s = this.ctx.status;
+    const { method, ctx: { status: s, response }, header } = this;
```

Comment:
Again, understanding what the syntax does is a much smaller part of it. The previous way clearly lines up variables assigned on the left and variables used on the right. Here, it's a lot more complicated, with renaming across multiple levels of destructuring. Sure, after taking a close look it becomes obvious what's assigned to what, but an initial glance might confuse a newly created variable with one destructured from `this`.